### PR TITLE
Create a MinGasAmount config parameter for chains

### PR DESCRIPTION
**Pull request by joeabbey**
## Description

This PR addresses https://github.com/cosmos/relayer/issues/867

## Checklist

- [x] I have made sure the upstream branch for this PR is correct
- [x] I have made sure this PR is ready to merge
- [x] I have made sure that I have assigned reviewers related to this project

### CODE REVIEW

**Comment by boojamya** [comment]
Nice troubleshooting! 
Super hacky... maybe we add better comment to explain why this is necessary.

File: client/tx.go [hunk 1]
```diff
@@ -285,6 +285,12 @@ type protoTxProvider interface {
 // BuildSimTx creates an unsigned tx with an empty single signature and returns
 // the encoded transaction or an error if the unsigned transaction cannot be built.
 func BuildSimTx(info *keyring.Record, txf tx.Factory, msgs ...sdk.Msg) ([]byte, error) {
+
+	// For estimating gas on evmos, let's use a terrible hack to set the gas to a fixed value
```
Comment (jackzampolin) [inline client/tx.go:289]: I don't like terrible hacks and would like to see a slightly different solution thats a bit more general. Maybe a `min-gas` config?

Comment (joeabbey) [reply client/tx.go]: Adding a MinGasAmount parameter

**Review by jtieri — APPROVED** [review]
Just to be clear, the idea here is that we will get this added to the chain registry and then this field will be configured automatically? I think we should open an issue on the relayer to provide some docs around this somewhere but otherwise this LGTM 🚀

**Comment by joeabbey** [comment]
> Just to be clear, the idea here is that we will get this added to the chain registry and then this field will be configured automatically? I think we should open an issue on the relayer to provide some docs around this somewhere but otherwise this LGTM 🚀

Right... 

Add to Lens, then add to Relayer, then add entries to Chain Registry... and future chain configs will just work.

I suspect we'll need to add this to any and all ethermint style chains until it changes behavior.

**Pull request merged**